log4j2/src/test/java/com/google/common/flogger/backend/log4j2/Log4j2Test.java

```diff
@@ -84,6 +92,15 @@ public void append(LogEvent event) {
 
   @Before
   public void setUpLoggerBackend() {
+    // need to reset logger config to prevent a clash with log4j2scopedloggingtest
+    final LoggerContext context = LoggerContext.getContext(false);
+    context.setConfiguration(new DefaultConfiguration());
+    context.updateLoggers();
+
+    System.getProperties().put(
```

Comment:
This shouldn't be necessary anymore since Flogger can now load the GrpcContextDataProvider automatically if it's on the classpath.